modules/core/react/lib/theming/theme.ts

```diff
@@ -0,0 +1,63 @@
+import colors from '@workday/canvas-colors-web';
+import deepmerge from 'deepmerge';
+import {CanvasTheme} from './types';
+
+/**
+ * Considerations:
+ * - Hex vs. Canvas colors
+ * - Hover/Active and constrasting text colors need to be calculateable. Two options:
+ *   1. Reverse lookup. i.e. given blueberry 400, we assume hover is 500 and active is 600. We still need to calculate colors outside our range though (i.e. Hex or if we run out of shades)
+ *   2. Calculate all colors. This would likely require a change to our colors, but would use the same approach in all cases.
+ * - Should consumers be able to theme grays? (e.g. pills, secondary buttons, etc.)
+ * - Should components that accept a color be controlled by the theme (e.g. StatusIndicator), or is it only errors and alerts?
+ * - Text colors?
+ * - Background?
+ */
+
+export const defaultCanvasTheme: CanvasTheme = {
+  palette: {
+    primary: {
+      light: colors.blueberry200,
+      main: colors.blueberry400,
+      dark: colors.blueberry500,
+      darkest: colors.blueberry600,
+      contrast: colors.frenchVanilla100,
+    },
+    alert: {
+      light: colors.cantaloupe300,
+      main: colors.cantaloupe400,
+      dark: colors.cantaloupe500,
+      darkest: colors.cantaloupe600,
+      contrast: colors.frenchVanilla100,
+    },
+    error: {
+      light: colors.cinnamon300,
+      main: colors.cinnamon500,
+      dark: colors.cinnamon600,
+      darkest: '#80160E',
+      contrast: colors.frenchVanilla100,
+    },
+    common: {
+      focusOutline: colors.blueberry400,
+    },
+  },
+  // Typography
+  // Breakpoints
+  // Depth?
+};
+
+/**
+ * Currently a work around for a default theme if no ThemeProvider exists.
+ * Tracked on https://github.com/emotion-js/emotion/issues/1193.
+ */
+export function getTheme(theme: Object): CanvasTheme {
```

Comment:
I ran into some problems with using a default theme if users don't wrap everything in a higher order `ThemeProvider` (you can track [here](https://github.com/emotion-js/emotion/issues/1193)). This is a slightly awkward work around for now, but not too bad.

Reply:
After thinking about how this could scale to products that are not fully built in React and only use some React components, I think this function will be required regardless of whether this issue is fixed. 

React contexts do not span multiple trees. Depending on how an app is structured, this could involve a lot of wrapping `CanvasProviders`. I figured a better solution was a single provider with `setThemeGlobal={true}`. This would set it on the `window` object and then `getTheme` looks for the provider theme first, if that doesn't exist, it checks on the window. Finally, it returns the default theme if nothing else is defined. Thoughts on this approach?